src/main/scala/com/microsoft/partnercatalyst/fortis/spark/StreamsChangeListener.scala

```diff
@@ -69,50 +69,62 @@ object StreamsChangeListener {
     }
 
     override def onMessageAsync(message: IMessage): CompletableFuture[Void] = {
-      logInfo(s"Service Bus message received ${message}.")
+      System.err.println(s"Service Bus message received ${message}.")
```

Comment:
Changing all these logInfo calls to System.err.println because we never want these messages to be turned off with log settings.